src/components/checkbox/Checkbox.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Tick from '../../icons/GlyphSmallTick'
+import './Checkbox.css'
+
+const Checkbox = ({className = '', label = '', disabled = false, onChange, ...props}) => {
+  className = `Checkbox pointer dib sans-serif ${className}`
+
+  const change = (event) => {
+    onChange(event.target.checked)
+  }
+
+  return (
+    <label className={className} {...props}>
+      <input type='checkbox' disabled={disabled} onChange={change} />
+      <span className='dib v-mid br1 w1 h1 mr1'>
+        <Tick className='w1 h1' fill='#69C4CD' viewBox='25 25 50 50' />
```

Comment:
We've got a `.fill-aqua` atom in [ipfs-css](https://github.com/ipfs-shipyard/ipfs-css/blob/ed7fa0fdae2c0b241ba1d2195614b843a0fea077/src/theme.css#L87)